wp-content/themes/access/src/js/modules/utility.js

```diff
@@ -279,8 +279,10 @@ Utility.webtrends = function(key, data) {
 
   let prefix = {};
   prefix['WT.ti'] = key;
-  data.unshift(prefix);
 
+  if (data !== 'undefined') {
```

Comment:
This comparison will not work because you are comparing against a string and not the type cast of the variable. The proper way to do this would be;

```
data !== undefined
```

or

```
typeof data !== 'undefined'
```

Additionally, because this conditional is only scoped for the `data.unshift(...` line any proceeding lines will also error out if data is `undefined`.

Preferably, this would be caught further upstream in the `Utility.track` function;

```
Utility.track = function(key, data) {
  // eslint-disable-next-line no-undef
  if (typeof Webtrends !== 'undefined') Utility.webtrends(key, data);

  return data;
};
```
By changing the conditional here;
```
if (typeof Webtrends !== 'undefined') Utility.webtrends(key, data);
```
to
```
if (typeof Webtrends !== 'undefined' || typeof data !== 'undefined') Utility.webtrends(key, data);
```
Then we can prevent the `Utility.webtrends` function from being fired at all. 

Bonus tip, there are other ways this could be handled as well, such as the `try/catch` statement https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Control_flow_and_error_handling#try...catch_statement

Reply:
@devowhippit thanks for this! I was unable to replicate the issue on FireFox. Good point made about handling upstream totally missed that.

Reply:
@theasteve you added the line to `Utility.webtrends` it needs to go in `Utility.track`

Reply:
@devowhippit should it be `&&` instead of `||`? Because in this case `data` can still be undefined if Webtrends is not since it would meet one of the conditions.

`if (typeof Webtrends !== 'undefined' || typeof data !== 'undefined') Utility.webtrends(key, data);`

Reply:
@theasteve yes you are right.